mne_bids/tests/test_utils.py

```diff
@@ -11,7 +11,15 @@
 from pathlib import Path
 
 from numpy.random import random
-import mne
+
+# This is here to handle mne-python <0.20
+import warnings
+with warnings.catch_warnings():
+    warnings.filterwarnings(action='ignore',
+                            message="can't resolve package",
+                            category=ImportWarning)
+    import mne
```

Comment:
you don't need noqa: F401 here?

Reply:
Good catch! _But_, in fact I have disabled F401 for all test files, as I would have needed to add `noqa: F401` to too many lines otherwise! See: https://github.com/mne-tools/mne-bids/blob/7bc36a46200c8436c27faf2df37e9e5b96d40593/setup.cfg#L4-L6

Reply:
No, `mne` is used a little further down in that file:
https://github.com/mne-tools/mne-bids/blob/7bc36a46200c8436c27faf2df37e9e5b96d40593/mne_bids/tests/test_utils.py#L35